src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -678,6 +687,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5037 = 5037,
 
+        /// <summary>
+        /// Empty icon extension error
+        /// </summary>
+        NU5044 = 5044,
+
+        /// <summary>
+        /// Invalid icon extension error
+        /// </summary>
+        NU5045 = 5045,
+
         /// <summary>
```

Comment:
"Invalid icon image file content" - We need to make sure file is actual image file somehow.

Reply:
Header validation at the beginning of the file is our option to avoid getting a dependency on WPF.

Reply:
Since this PR is only for icon filename extension, I will address file content validation in another PR: https://github.com/NuGet/Home/issues/10235